SPECS/cni-plugins/cni-plugins.spec

```
@@ -39,8 +39,14 @@ make -k check |& tee %{_specdir}/%{name}-check-log || %{nocheck}
 %{_default_cni_plugins_dir}/*
 
 %changelog
+<<<<<<< HEAD
 * Tue Oct 10 2023 Dan Streetman <ddstreet@ieee.org> - 0.9.1-15
 - Bump release to rebuild with updated version of Go.
+=======
+* Wed Oct 11 2023 Mateusz Gozdek <mgozdek@microsoft.com> - 1.3.0-1
+- Make plugin binaries correctly print version
+- Upgrade to version 1.3.0
+>>>>>>> 5c54ee34c (SPECS/cni-plugins: update to v1.3.0 and set version while building)
```

```suggestion
* Wed Oct 11 2023 Mateusz Gozdek <mgozdek@microsoft.com> - 1.3.0-1
- Make plugin binaries correctly print version
- Upgrade to version 1.3.0

* Tue Oct 10 2023 Dan Streetman <ddstreet@ieee.org> - 0.9.1-15
- Bump release to rebuild with updated version of Go.
```

I believe this suggestion should work